src/utils/__tests__/compare.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import * as sut from '../compare'
+
+// ----------------------------------------------------------------------------
+// Tests
+describe( 'arrayEquals', () => {
+  it( 'compares different length arrays', () => {
+    const res = sut.arrayEquals( [], [ 1, 2 ] )
+    expect( res ).toBeFalsy()
+  } )
+
+  it( 'compares equal arrays', () => {
+    const res = sut.arrayEquals( [ 1, 2 ], [ 1, 2 ] )
```

Comment:
Does element order matter?

Reply:
yes. I just added another test to verify a different order.